src/test/java/uk/gov/pay/adminusers/queue/event/EventMessageHandlerTest.java

```diff
@@ -164,6 +164,9 @@ void shouldHandleDisputeCreatedEvent() throws QueueException {
         assertThat(personalisation.get("disputeEvidenceDueDate"), is("7 March 2022"));
         assertThat(personalisation.get("sendEvidenceToPayDueDate"), is("4 March 2022"));
 
+        assertThat(personalisation.get("fraudulent"), is("yes"));
```

Comment:
Is it worth adding assertions to this test that all the other reasons are present with value "no"?

Reply:
I'll do it when removing the unused ones 👍